fastchat/serve/api_provider.py

```diff
@@ -129,9 +140,21 @@ def openai_api_stream_iter(
         model_name = "gpt-4-1106-preview"
 
     # Make requests
+    text_messages = []
+    for message in messages:
+        if type(message["content"]) == str:  # text-only model
+            text_messages.append(message)
+        else:  # vision model
+            filtered_content_list = [
+                content for content in message["content"] if content["type"] == "text"
```

Comment:
```suggestion
                content for content in message["content"]
```
Why filter out the non-text fields?

Reply:
The main idea was to not log the base64 images. When we take an image from gradio, we convert it to base64 and then place it into the messages. Before, we would call `logger.info` with these messages, however this means that we would log down entire images, which can accumulate a lot of space. By filtering for only text messages, we no longer log the images to save space.